specification/logs/semantic_conventions/media.md

```diff
@@ -29,19 +29,30 @@ As such, these should be recorded as Log Record attributes when applicable. They
 
 **Description:** A file to which log was emitted.
 
-| Name                            | Notes and examples                                                                       |
-| ------------------------------- | ---------------------------------------------------------------------------------------- |
-| `log.file.name`          | The basename of the file. Example: `audit.log`                                           |
-| `log.file.path`          | The full path to the file. Example: `/var/log/mysql/audit.log`                           |
-| `log.file.name_resolved` | The basename of the file, with symlinks resolved. Example: `uuid.log`                    |
-| `log.file.path_resolved` | The full path to the file, with symlinks resolved. Example: `/var/lib/docker/uuid.log`   |
+<!-- semconv log.file -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `log.file.name` | string | The basename of the file. | `audit.log` | Opt-In |
+| `log.file.path` | string | The full path to the file. | `/var/log/mysql/audit.log` | Opt-In |
+| `log.file.name_resolved` | string | The basename of the file, with symlinks resolved. | `uuid.log` | Opt-In |
+| `log.file.path_resolved` | string | The full path to the file, with symlinks resolved. | `/var/lib/docker/uuid.log` | Opt-In |
```

Comment:
In Collector's [filelog](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver) receiver `log.file.name` is included by default and the rest are opt-in. I suggest we make `log.file.name` Recommended here.

Reply:
@ChrsMark I think you need to regenerate the md file. I can see the change in yaml but not in md.